backend/protocol_rpc/message_handler/fastapi_handler.py

```
@@ -0,0 +1,196 @@
+import copy
+import asyncio
+from functools import wraps
+import traceback
+from typing import Optional, Any
+
+from loguru import logger
+import sys
+
+from backend.protocol_rpc.message_handler.types import LogEvent
+from backend.protocol_rpc.configuration import GlobalConfiguration
+from backend.protocol_rpc.message_handler.types import EventScope, EventType, LogEvent
```

Ruff (0.13.3)

Redefinition of unused `LogEvent` from line 12

Remove definition: `LogEvent`

(F811)